include/deal.II/base/index_set.h

```diff
@@ -223,6 +223,11 @@ class IndexSet
   unsigned int n_intervals () const;
 
   /**
+   * This function returns the local index of the beginning of the largest range.
+   */
+  unsigned int largest_range_index() const;
```

Comment:
Actually, reading through the implementation, it isn't quite clear to me what the function returns. I think it returns the `index_within_set` of the first index in the largest range, right? If so, then the name of the function is not well chosen, nor is the comment that describes it. How about calling it `starting_index_within_set_of_largest_range()`? That's a bit of a mouthful, but presumably few people will call this function. 

Alternative names are welcome.